stringfuzz/transformers/stem_cut.py

```diff
@@ -0,0 +1,128 @@
+'''
+The stem-cut transform picks a subtree and a leaf at random
+and swaps them for each type. Almost like cutting a stem and planting it 
+somewhere else. Names are hard.
```

Comment:
Would the word "graft" be suitable? [Grafting](https://en.wikipedia.org/wiki/Grafting) is when a part of one plant (usually tree) is cut off and attached to another.